Code/GraphMol/ROMol.cpp

```diff
@@ -102,6 +102,17 @@ void ROMol::initFromOther(const ROMol &other, bool quickCopy, int confId) {
     d_stereo_groups.emplace_back(otherGroup.getGroupType(), std::move(atoms));
   }
 
+  if (other.dp_delAtoms) {
+    dp_delAtoms.reset(new boost::dynamic_bitset<>(*other.dp_delAtoms));
```

Comment:
I'm not sure I would copy over the editing state.  This seems really dangerous.

Reply:
I don't see how there's any danger here given that no changes have actually been made to the molecule.